index.html

```diff
@@ -231,6 +230,33 @@ <h3>Languages</h3>
         <img src="assets/vector.svg " alt="">
       </div>
   </section>
+
+  <div class="container-white">
+ 
+    <section class="contact-me">
+      
+      <form action="https://formspree.io/f/xbjevbkv" method="POST">
+        <div class="background">
+        <h2>contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+          I’d love to help with it!</p>
+        <label for="name">
+          <input type="text" id="name" placeholder="Lorem Ipsum" required maxlength="30">
+        </label>
+        <label for="email">
+          <input type="email" id="message" placeholder="loremipsum@gmail.com" required/>
+        </label>
+        <label for="message-box">
+          <textarea id="message-box" placeholder="Please enter your message here" required maxlength="500"></textarea>
+        </label>
+        <button type="submit" class="el-botones" name="submit">
+          <p>Get in touch</p>
+        </button>
```

Comment:
- [ ] [ OPTIONAL ] Kindly change the **p** tag to **span** instead. Your code is technically valid and will work in most modern browsers 💻 🖥️ 📱 However, it's worth noting that using a **p** element directly inside a **button** element is not considered best practice in HTML.
Best practice suggests that for button labels, you should use a more appropriate **inline-level element** such as <span> or simply plain text. Here's an example: 👇🏽 👇🏽 

```sh
<button type="submit" class="el-botones" name="submit">
  <span>Get in touch</span>
</button>
```